clients/models/metadata.go

```diff
@@ -30,11 +33,6 @@ type Metadata struct {
 
 // Validate validates this metadata
 func (m *Metadata) Validate(formats strfmt.Registry) error {
-	var res []error
-
-	if len(res) > 0 {
```

Comment:
Why this validation was removed?

Reply:
it's an autogenerated var that checks nothing since in this case there's nothing to validate.